.scrutinizer.yml

```diff
@@ -0,0 +1,12 @@
+filter:
+    excluded_paths: [tests/*]
+build:
+    tests:
+        override:
+            -
+                command: 'vendor/bin/phpunit --coverage-clover=coverage.clover'
```

Comment:
Since Scrutinizer doesn't support this, the separate PHP version checks are now performed using Travis.